sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateAsyncClientTest.java

```diff
@@ -780,23 +736,15 @@ public void listCertificateIssuers(HttpClient httpClient, CertificateServiceVers
         listCertificateIssuersRunner((certificateIssuers) -> {
             HashMap<String, CertificateIssuer> certificateIssuersToList = new HashMap<>(certificateIssuers);
 
-            for (CertificateIssuer issuer : certificateIssuersToList.values()) {
-                CertificateIssuer certificateIssuer = certificateAsyncClient.createIssuer(issuer).block();
-
-                assertTrue(issuerCreatedCorrectly(issuer, certificateIssuer));
-            }
-
             List<IssuerProperties> output = new ArrayList<>();
 
-            StepVerifier.create(certificateAsyncClient.listPropertiesOfIssuers()
-                .map(issuerProperties -> {
-                    output.add(issuerProperties);
-                    return Mono.empty();
-                })).assertNext(ignore -> assertEquals(certificateIssuersToList.size(), output.size()));
-
-            for (CertificateIssuer issuer : certificateIssuers.values()) {
-                certificateAsyncClient.deleteIssuer(issuer.getName()).block();
-            }
+            StepVerifier.create(Flux.fromIterable(certificateIssuers.values())
+                    .flatMap(issuer -> certificateAsyncClient.createIssuerWithResponse(issuer)).last().map(ignored -> certificateAsyncClient.listPropertiesOfIssuers()
+                        .map(issuerProperties -> {
+                            output.add(issuerProperties);
+                            return Mono.empty();
+                        })))
+                .assertNext(ignore -> assertEquals(certificateIssuersToList.size(), output.size()));
```

Comment:
The recording for this test turned out empty because we are not waiting for the async operation to complete.
```suggestion
                .assertNext(ignore -> assertEquals(certificateIssuersToList.size(), output.size()))
                .verifyComplete();
```